src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -2667,6 +2667,56 @@ void java_lang_Throwable::get_stack_trace_elements(Handle throwable,
   }
 }
 
+Handle java_lang_Throwable::get_cause_with_stack_trace(Handle throwable, TRAPS) {
+  // Call to JVM to fill in the stack trace and clear declaringClassObject to
+  // not keep classes alive in the stack trace.
+  // call this:  public StackTraceElement[] getStackTrace()
+  assert(throwable.not_null(), "shouldn't be");
+
+  JavaValue result(T_ARRAY);
+  JavaCalls::call_virtual(&result, throwable,
+                          vmClasses::Throwable_klass(),
+                          vmSymbols::getStackTrace_name(),
+                          vmSymbols::getStackTrace_signature(),
+                          CHECK_NH);
+  Handle stack_trace(THREAD, result.get_oop());
+  assert(stack_trace->is_objArray(), "Should be an array");
+
+  // If the original exception was in java.base, then use that otherwise use
```

Comment:
comment is about one specific bootstrap module (java.base) but the check is for bootstrap loader.

Reply:
Ok, thanks for pointing that out. I fixed the comment.  I realized that really wanted the bootloader after I wrote the comment.